src/lib/javascript.d.ts

```diff
@@ -1948,6 +1948,7 @@ declare global {
 
         interface TypedState<T extends iobJS.StateValue = any> extends iobJS.State {
             val: T;
+            notExist?: true;
```

Comment:
Are you sure this value is `true`?

Reply:
Even in the "existing state" case? Interesting.

Reply:
IN existing state it is undefined